src/NetCDF.jl

```diff
@@ -369,6 +369,17 @@ function putatt(ncid::Integer,varid::Integer,atts::Dict)
     end
 end
 
+function putatt(nc::NcFile, atts::Dict)
+    putatt(nc.ncid, NC_GLOBAL, atts)
+    merge!(nc.gatts, getatts(nc.ncid,NC_GLOBAL,collect(keys(atts))))
+end
+
+function putatt(nc::NcFile, var::NcVar, atts::Dict)
+    @assert var===nc.vars[var.name]
```

Comment:
Given that this check is necessary, should the function rather have the signature `function putatt(var::NcVar, atts::Dict)`? The NetCDF id can be obtained from the var.ncid field.

Reply:
I agree, I've updated the PR.